tests/test_draw/test_draw_integration.py

```diff
@@ -75,6 +76,64 @@ def test_domain_element_size(self):
         with self.assertRaises(ValueError):
             x = get_domain(2, 2, is_y=False)
 
+class DrawableObjectTests(unittest.TestCase):
+    """testing Drawable object methods and properties"""
+
+    def test_traces(self):
+        """test trace initialisation"""
+        d = Drawable()
+        self.assertEqual(d.traces, [])
+        trace = dict(type="scatter", x=[0, 1], y=[0, 1])
+        d = Drawable(traces=[trace])
+        self.assertEqual(d.traces, [trace])
+
+    def test_add_traces(self):
+        """test trace add method"""
+        d = Drawable()
+        self.assertEqual(d.traces, [])
+        trace = dict(type="scatter", x=[0, 1], y=[0, 1])
+        d.add_trace(trace)
+        self.assertEqual(d.traces, [trace])
+
+    def test_bound_to(self):
+        """bound object should have the drawable object and show methods"""
+        class TestObj:
+            pass
+        
+        o = TestObj()
+        d = Drawable()
+        b = d.bound_to(o)
+        self.assertEqual(b.drawable, d)
+        self.assertTrue(getattr(b, "iplot", False))
+        self.assertTrue(getattr(b, "show", False))
```

Comment:
Change `getattr` to `hasattr`, e.g. `getattr(b, "iplot", False)` to `hasattr(b, "iplot")`